locallibrary/catalog/admin.py

```diff
@@ -1,3 +1,36 @@
 from django.contrib import admin
+from .models import Author, Genre, Book, BookInstance
 
 # Register your models here.
+admin.site.register(Genre)
+
+# Define the admin class
+class AuthorAdmin(admin.ModelAdmin):
```

Comment:
dùng đồng nhất cách register site admin nha
Line 8-> 13
```suggestion
@admin.register(Author)
class AuthorAdmin(admin.ModelAdmin):
    ...
```